hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.3__index_init.sql

```diff
@@ -149,7 +149,7 @@ create index if not exists nft_allowance_history__timestamp_range on nft_allowan
 
 -- nft_transfer
 create index if not exists nft_transfer__timestamp on nft_transfer (consensus_timestamp desc);
-create unique index if not exists nft_transfer__token_id_serial_num_timestamp
+create index if not exists nft_transfer__token_id_serial_num_timestamp
     on nft_transfer (token_id desc, serial_number desc, consensus_timestamp desc, payer_account_id);
```

Comment:
remove `payer_account_id` from the column
```suggestion
create index if not exists nft_transfer__token_id_serial_num_timestamp
    on nft_transfer (token_id desc, serial_number desc, consensus_timestamp desc);
```

Reply:
Done.